index.html

```diff
@@ -108,12 +108,16 @@ <h3>
             Click for Demo
         </button>
 
-        <div>
+        <div class="fav-panel">
             <br>
-            <button class="btn" id="focus-btn">
+            <button class="btn " id="focus-btn">
                 <i class="fa fa-headphones"></i>
                 Focus
             </button>
+            <button class="btn " id="lofi-btn">
```

Comment:
I didn’t notice this before, can we remove the extra space here and on line 113? I know the other one is old.

Reply:
Hi @shakeelmohamed, changes are done. Please check now